tests/autorestart_test/test_container_autorestart.py

```diff
@@ -0,0 +1,327 @@
+"""
+Test the auto-restart feature of containers 
+"""
+import time
+import pytest
+import logging
+
+from collections import defaultdict
+
+from common.utilities import wait_until
+from common.helpers.assertions import pytest_assert
```

Comment:
Import has to be changed to from tests.common.helpers.assertions import pytest_assert

Reply:
Thanks for suggestion! Updated.